src/interface/readconfigfile.py

```diff
@@ -0,0 +1,115 @@
+import xml.etree.ElementTree as etree
+class ReadConfigFile():
```

Comment:
the name of the file should be separated with underscores i.e. read_config_file.py